src/usr/local/www/firewall_rules_edit.php

```diff
@@ -1090,6 +1084,17 @@ function build_if_list() {
 	return($iflist);
 }
 
+// Expands missing/blank/"all" icmptypes to a list of specific types. Doesn't check validity.
+function expand_icmptype($icmptype, $ipprotocol) {
+	if (isset($icmptype) && is_string($icmptype) && strlen($icmptype) > 0 && $icmptype != "any") {
```

Comment:
$icmptype will always be set since it's a parameter, this isset() call is unnecessary.